Clarified versions available in deployment UI and CLI (#981)
kubeflow/website: content/docs/pipelines/tutorials/pipelines-tutorial.md

  guide to [deploying Kubeflow on GCP](/docs/gke/deploy/), 
  taking note of the following:

>   * For the simplest deployment experience, use the Kubeflow deployment web app

Reviewer: ```suggestion
  * If you want the most simple deployment experience, use the Kubeflow deployment web app
```

    as described in the guide to 

>     [deployment using the UI](/docs/gke/deploy/deploy-ui/).

Reviewer: ```suggestion
    [deploy Kubeflow using the UI](/docs/gke/deploy/deploy-ui/).
```

Author: Same comment as above.

    The deployment web app currently supports 
    **Kubeflow {{% kf-deployment-ui-version %}}**.
  * For more control
    over the deployment, use the guide to 

>     [deployment using the CLI](/docs/gke/deploy/deploy-cli/).

Reviewer: ```suggestion
    [deploy Kubeflow using the CLI](/docs/gke/deploy/deploy-cli/).
```

Author: Same reply as above.
